### PR TITLE
Fixes #21 save changes model. Also fixes #118 for now

### LINKED ISSUE #21: Basic UseCase: save change model
- load model

- change it, e.g.change parameter values

- save the model (preferable in a format which can be loaded in PK-Sim/MoBi)

### LINKED ISSUE #118: Move functions operating on simulation to utilities-simulation
I propose to move the functions `load-simulation` and `run-simulation` to the script called `utilities-simulation` to be consistent with container and parameter.

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: R/error-checks.R [hunk 1]
```diff
@@ -30,23 +30,25 @@ isSameLength <- function(...) {
 
 
 validateIsOfType <- function(object, type) {
-  if (!isOfType(object, type)) {
```
[the remark this answers is not in the document]
Comment (PavelBal) [reply R/error-checks.R]: True.

**Comment by PavelBal** [comment]
I am getting an error every time when trying to get `simulationPersister`:

```
simulationPersister <- getNetTask("SimulationPersister")
Error in clrCallStatic("OSPSuite.R.Api", paste0("Get", taskName)) : 
  Type:    System.MissingMethodException
Message: Could not find a suitable static method GetSimulationPersister on type OSPSuite.R.Api without method parameters
Method:  Void ThrowMissingMethod(System.Type, System.String, System.String, System.Type[])
Stack trace:
   bei Rclr.ReflectionHelper.ThrowMissingMethod(Type classType, String methodName, String modifier, Type[] types)
   bei Rclr.ClrFacade.InternalCallStaticMethod(Type classType, String methodName, Boolean tryUseConverter, Object[] arguments)
   bei Rclr.ClrFacade.CallStaticMethod(String typename, String methodName, Object[] arguments)
```

However, all checks are working (including the tests)... Do you have an idea, why?

**Comment by msevestre** [comment]
Make sure you uninstall ospsuite from your package lib. I would remove the .rdata as well

**Comment by msevestre** [comment]
ok going to merge this to implement the SimulationSettings and output stuff. I'd like to avoid merge conflicts as much as possible

**Pull request merged**